CONTRIBUTING.md

```diff
@@ -57,6 +57,81 @@ You can build and start a local docs website by running these two commands.
   - `npm run docs-buildDockerImage` OR `docker build -t squidfunk/mkdocs-material ./docs/`
   - `npm run docs-runLocalDocker` OR `docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material`
 
+### Tests
+
+Tests are under `tests` folder of each modules and split into two categories: unit tests and e2e tests.
```

Comment:
```suggestion
Tests are under `tests` folder of each modules and split into two categories: unit tests and e2e (end-to-end) tests.
```

Nit: both me and you know what this means, but it would be good to clarify to make it inclusive for folks with different level of proficiency

Reply:
I do see it explained down in the page though, so that's nice!